controller/src/main/java/org/jboss/as/controller/ControlledProcessState.java

```diff
@@ -38,10 +38,18 @@
          */
         STARTING("starting"),
         /**
+         * @deprecated see {@link #OK}
+         * @see {@link org.jboss.as.controller.operations.common.ProcessStateAttributeHandler}
+         * for how the new values are mapped to the old attributes for compatibility.
          * The process is started, is running normally and has a runtime state consistent with its persistent configuration.
          */
         RUNNING("running"),
         /**
+         * replaces #RUNNING
+         * The process is started, is operating normally and has a runtime state consistent with its persistent configuration.
+         */
+        OK("ok"),
```

Comment:
Ok, thats good to know. What about removing that from the enum and just mapping runtime-configuration-state in the attribute handler, but leaving everything else the same? That would allow us to introduce the new attribute, but not break anything until the next major.